acceptance/terrafarm/farmer.go

```diff
@@ -271,6 +281,26 @@ func (f *Farmer) Assert(t *testing.T) {
 	}
 }
 
+// AssertStates verifies that the cluster state matches the provided map
+// of process names (e.g. cockroach) to expected states (e.g. RUNNING).
+func (f *Farmer) AssertStates(t *testing.T, procToExpState map[string]string) {
```

Comment:
What if something is expected to be down? After the load test period is over, `block_writer` should exit normally.